documentation/source/overview/studies.rst

```diff
@@ -5,96 +5,96 @@ Studies using SCT
 
```

Comment:
One very minor, nitpicky comment for future reference: If you name your branch `sy/improve_reference` instead, some IDEs (like PyCharm) will visually group your branches together under a folder called `sy`. 

Right now, since it's `improve_reference/sy`, the folder displays as `improve_reference` instead. :stuck_out_tongue_closed_eyes: 

![Screenshot from 2021-08-12 18-13-56](https://user-images.githubusercontent.com/16181459/129276784-fe63b0c7-8a83-4571-8abd-39097026b687.png)

Reply:
Yeah, this was my bad. I usually  branches like `sy/<branch_name>` but for this one wrote t mistakenly in wrong format. Will be careful next time :)